src/python/pants/backend/python/pants_requirement.py

```diff
@@ -50,7 +50,9 @@ def __call__(self, name=None, dist=None):
                                       msg='The {} target only works for pantsbuild.pants '
                                           'distributions, given {}'.format(self.alias, dist))
 
-    env_marker = "python_version>='3.4' or (python_version>='2.7' and python_version<'3')"
+    # Update the environment marker in lockstep with other changes as described in
```

Comment:
Is it worth having these comments refer to one another by suggesting looking for other instances of `https://github.com/pantsbuild/pants/issues/6450` in the codebase?

Reply:
I don't think so - the comments already fan into the issue which has pointers to the three code locations.